src/diffpy/labpdfproc/functions.py

```diff
@@ -172,9 +182,9 @@ def get_path_length(self, grid_point, angle):
         return total_distance, primary_distance, secondary_distance
 
 
-def compute_cve(diffraction_data, mud, wavelength):
+def compute_cve(diffraction_data, mud, wavelength, brute_force=False):
```

Comment:
My idea was that we have different cve computers, so we don't need to pass `brute_force=false`.  We either call `fast_cve` or `brute_force_cve`, depending on whether brute_force is true or not, but the choice is handled in main.

both calculators need to have the same signatures.